homeassistant/components/tradfri/sensor.py

```diff
@@ -96,11 +97,32 @@ def _get_filter_time_left(device: Device) -> int:
         native_unit_of_measurement=TIME_HOURS,
         icon="mdi:clock-outline",
         value=_get_filter_time_left,
-        unique_id_suffix=slugify(ATTR_FILTER_LIFE_REMAINING),
     ),
 )
 
 
+def _migrate_old_unique_ids(hass: HomeAssistant, old_unique_id: str, key: str) -> None:
+    """Migrate unique IDs to the new format."""
+    ent_reg = entity_registry.async_get(hass)
+
+    if entity_id := ent_reg.async_get_entity_id(Platform.SENSOR, DOMAIN, old_unique_id):
```

Comment:
Invert this check and `return` if true. Then we can outdent below.

Reply:
Not quite sure I follow you here but we now return if the deprecated unique id is not found.